gdg_events/google_io_extended.md

```diff
@@ -1,6 +1,6 @@
-# Google I/O Extended 2015
+# Google I/O Extended 201
 
-![](https://events.google.com/io2015/images/io15-color.png)
+![](http://www.androidauthority.com/wp-content/uploads/2016/02/Google-IO-2016.jpg)
```

Comment:
Could you change the url to the uploaded image (https://github.com/gdg-wisdom/wisdom-2016/raw/master/Google-IO-2016.jpg)?